src/integrationTest/java/com/mongodb/hibernate/BasicCrudIntegrationTests.java

```diff
@@ -45,13 +46,14 @@
         annotatedClasses = {
             BasicCrudIntegrationTests.Book.class,
             BasicCrudIntegrationTests.BookWithEmbeddedField.class,
+            BasicCrudIntegrationTests.BookDynamicallyInserted.class,
             BasicCrudIntegrationTests.BookDynamicallyUpdated.class
         })
 @ExtendWith(MongoExtension.class)
 class BasicCrudIntegrationTests implements SessionFactoryScopeAware {
 
     @InjectMongoCollection("books")
-    private static MongoCollection<BsonDocument> collection;
+    private static MongoCollection<BsonDocument> mongoCollection;
```

Comment:
to maintain consistency with other integration testing classes (both `MongoStatementIntegrationTests` and `MongoPreparedStatementIntegrationTests` use `mongoCollection`)